src/igvfd/types/file_set.py

```diff
@@ -86,6 +86,35 @@ class MeasurementSet(FileSet):
     item_type = 'measurement_set'
     schema = load_schema('igvfd:schemas/measurement_set.json')
 
+    @calculated_property(
+        condition='multiome_size',
+        schema={
+            'title': 'Related Multiome Datasets',
+            'description': 'Related datasets included in the multiome experiment this measurement set is a part of.',
+            'type': 'array',
+            'uniqueItems': True,
+            'items': {
+                'title': 'Related Multiome Dataset',
+                'description': 'Related dataset included in the multiome experiment this measurement set is a part of.',
+                'type': 'string',
+                'linkTo': 'MeasurementSet'
+            },
+            'notSubmittable': True,
+        }
+    )
+    def related_multiome_datasets(self, request, accession, samples=None):
+        if samples:
+            related_datasets = []
+            for sample in samples:
+                sample_object = request.embed(sample, '@@object')
+                if sample_object.get('file_sets'):
+                    for file_set in sample_object.get('file_sets'):
+                        if 'measurement-sets' in file_set and \
+                            accession not in file_set and \
+                                file_set not in related_datasets:
+                            related_datasets.append(file_set)
```

Comment:
it took me a while to understand, but I think I do understand now. (1) I think for clarity "file_set" has to be renamed into "file_set_id". (2) I think that "replaced" objects loose their accession, so one cannot rely on the presence of accession, instead use @id, (3) probably good idea also to indicate the measurment-sets is not just part of the id , but specifically prefix.